src/main/java/uk/gov/hmcts/reform/jobscheduler/Application.java

```diff
@@ -3,11 +3,9 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.cloud.client.circuitbreaker.EnableCircuitBreaker;
-import org.springframework.cloud.netflix.feign.EnableFeignClients;
 import org.springframework.cloud.netflix.hystrix.dashboard.EnableHystrixDashboard;
 
 @SpringBootApplication
-@EnableFeignClients
```

Comment:
can we also remove the `spring-cloud-starter-openfeign` dependency from build.gradle?

Reply:
all libs come from `service-auth-provider-client` - should be good to remove. planned to do so too